src/style.css

```diff
@@ -1 +1,20 @@
-/* styles go here */
+.input__container {
+  margin-bottom: 20px;
+}
+
+.input__container:last-child {
+  margin-bottom: 0;
+}
+
+.input___label {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.input___label:last-of-type {
```

Comment:
```suggestion
.input___label:last-child {
```
try this
<img width="572" alt="image" src="https://user-images.githubusercontent.com/66749919/200026115-23b36fc7-8f30-4f37-b5a0-d341d432a004.png">

Reply:
or maybe you didn`t deploy your changes?

Reply:
sorry, i didn't deploy. last-of-type works